instrument/loq/sans.py

```diff
@@ -14,7 +14,11 @@ class LOQ(ScanningInstrument):
     def __init__(self):
         super().__init__()
         self.setup_sans = self.setup_dae_histogram
-        self._poslist_dls = self.get_pv("LKUP:DLS:POSITIONS").split()
+        try:
+            self._poslist_dls = self.get_pv("LKUP:DLS:POSITIONS").split()
+        except AttributeError:
+            warning("No positions found for DLS Sample Changer!")
```

Comment:
Does this mean that if they change configs into a config with the DLS  sample changer, they then need to reload these scripts? If so that's a bit unintuitive?

Would it be better for `_poslist_dls` to be a function that's called when-needed rather than in `__init__`?